modules/promotions/admin-menu-items/custom-icons-promotion-item.php

```diff
@@ -22,7 +22,7 @@ public function get_page_title() {
 	protected function get_promotion_title():string {
```

Comment:
We shouldn't have any unrelated changes in our PR. Can you please try to pull the latest elementor/main branch?